### PR TITLE
Add more examples in the gallery

### LINKED ISSUE #13364: Add more examples in the gallery
### Problem description

During the course of the project on *[Fundamentals of data visualization](https://github.com/bokeh/dataviz-fundamentals/tree/main)*, @bryevdv suggested we add some of the plots to our gallery examples and docs. A good example to add would be the `SLOPE GRAPH` found in this [notebook](https://github.com/bokeh/dataviz-fundamentals/blob/main/notebooks/06-Visualizing-associations-among-variables.ipynb). 

### Feature description

Here's the image of the plot:image

Since the [csv file](https://github.com/bokeh/dataviz-fundamentals/blob/main/data/csv_files/CO2_emissions.csv) for this plot is a small file (~ 300kb), I suggest we also add it as part of the source files as well.

@bokeh/dev

### PR DESCRIPTION
**Pull request by Azaya89**
- [x] ref: #13364 
- [x] tests added / passed

### CODE REVIEW

File: src/bokeh/sampledata/emissions.py
Comment (bryevdv) [inline src/bokeh/sampledata/emissions.py]: @Azaya89 I would stop here, more or less :) Doing all the processing below just precludes other users from using the full dataset in case they would like to do that.  If there are simple type-conversions or other cleanup, that could be ok, but I would return a full dataframe for all raw data and leave it to examples or users to do any filtering or joining, etc.

Comment (Azaya89) [reply src/bokeh/sampledata/emissions.py]: OK

File: examples/topics/categorical/slope_graph.py
Comment (mattpap) [inline examples/topics/categorical/slope_graph.py]: This should use `Text` glyph instead.

File: src/bokeh/sampledata/emissions.py
Comment (mattpap) [inline src/bokeh/sampledata/emissions.py]:
```suggestion
    df = package_csv('emissions', 'emissions.csv')
```

File: src/bokeh/sampledata/emissions.py
Comment (mattpap) [inline src/bokeh/sampledata/emissions.py]:
```suggestion
    countries = [
        "Trinidad and Tobago",
        "Qatar",
        "United Arab Emirates",
        "Oman",
        "Bahrain",
        "Singapore",
        "Netherlands Antilles",
        "Kazakhstan",
        "Equatorial Guinea",
        "Kuwait",
    ]
```

**Comment by bryevdv** [comment]
@Azaya89 I have uploaded the file to s3 as `emmissions.csv`, it just needs to be added to `util/sampledata.json` with a hash. You can see how the hashes are computed for comparison in `util/sampledata.py` (let me know any questions!)

**Comment by Azaya89** [comment]
@mattpap @bryevdv , is there something pending I need to do to progress this PR?

**Comment by bryevdv** [comment]
@Azaya89 we will need thumbnails for the gallery. I will generate some with the exact dimensions and image optimization that has been used for all the existing ones, and attach them here as a comment.

**Comment by Azaya89** [comment]
OK. Thank you!

**Comment by bryevdv** [comment]
@Azaya89 I forgot it would be a bit of a chore to run this locally until the sampledata changes are merged. Can we do this instead?

* remove the gallery JSON entry
* merge with the sampledata and docs changes (now its easy to run on `branch-3.3`)
* add gallery in a separate PR

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @Azaya89 !

**Pull request merged**
